File: Repl.it/Python/Распаковка_строки.py
print("Введите строку необходимую распаковать")
s = str(input())
n = int(0)
while len(s) != 0:
    n = (int)(s[0] - "0")
    for j in (0, n, 1):
        print(s[1])
    s.erase(0, 2)

File: Repl.it/C++/Распаковка_строки.cpp
#include <iostream>
#include <string>
using namespace std;
int main()
{
    cout << ("Введите строку необходимую распаковать");
    string s;
    int n = 0;
    cin >> s;
    while (sizeof(s) != 0)
    {
        n = (int)(s[0] - '0');
        for (int j = 0; j < n; j++)
            cout << (s[1]);
        s.erase(0, 2);
    }
}
